File: ISAProject/src/main/java/com/app/bidder/BidderService.java
package com.app.bidder;

import com.app.offers.Offer;
import com.app.restaurant.Restaurant;

public interface BidderService {
	
	Iterable<Bidder> findAll();
	Bidder findOne(String email);
	Bidder save(Bidder bidder);
	void delete(String email);
	Iterable<Bidder> findByRestaurant(Restaurant restaurant);
}

File: ISAProject/src/main/webapp/bidder/bidderService.js
app.service('bidderService', ['$http', function($http){

    var url='/bidders';
	
	this.findAll = function(restaurant){
		return $http.get(url+"/findAll/"+restaurant);
	}
    this.findOne = function(email){
		return $http.get(url + "?email=" + email);
	}
    this.update = function(waiter){
		return $http.put(url, waiter);
	}
    this.save = function(waiter){
		return $http.post(url, waiter);
	}    
     
}]);